api/auth/validate.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2023 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package auth
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/gin-gonic/gin"
+	"github.com/go-vela/server/router/middleware/claims"
+	"github.com/go-vela/server/util"
+)
+
+// swagger:operation GET /validate-token authenticate ValidateServerToken
+//
+// Validate a server token
+//
+// ---
+// produces:
+// - application/json
+// security:
+//   - CookieAuth: []
+// responses:
+//   '200':
+//     description: Successfully validated a token
+//     schema:
+//       type: string
+//   '401':
+//     description: Unauthorized
+//     schema:
+//       "$ref": "#/definitions/Error"
+
+// ValidateServerToken will return the claims of a valid server token
+// if it is provided in the auth header.
```

Comment:
```suggestion
// ValidateServerToken will validate if a token was issued by the server
// if it is provided in the auth header.
```

This was my mistake when writing this endpoint, and I'm just now seeing it... 😅